src/main/java/randoop/generation/InputSequenceSelector.java

```diff
@@ -1,15 +1,27 @@
 package randoop.generation;
 
+import randoop.sequence.ExecutableSequence;
 import randoop.sequence.Sequence;
 import randoop.util.SimpleList;
 
 /** Interface for selecting sequences as input for creating new sequences. */
-public interface InputSequenceSelector {
+public abstract class InputSequenceSelector {
   /**
    * Choose a sequence used as input for creating a new sequence.
    *
    * @param candidates sequences to choose from
    * @return the chosen sequence
    */
-  Sequence selectInputSequence(SimpleList<Sequence> candidates);
+  public abstract Sequence selectInputSequence(SimpleList<Sequence> candidates);
+
+  /**
+   * A hook that is called after a new sequence has been created and executed.
+   *
+   * <p>The default implementation does nothing. Subclasses may override it to add behavior.
+   *
+   * @param eSeq the recently executed sequence which is new and unique, and has just been executed.
+   *     It contains its overall execution time which can be associated with the underlying {@link
+   *     Sequence} in {@code eSeq}.
+   */
+  public void createdExecutableSequenceFromInputs(ExecutableSequence eSeq) {}
```

Comment:
You removed the inputs from this, so the name should no longer contain `fromInputs`.